sources/packages/web/src/views/institution/locations/offerings/OfferingRequestChange.vue

```diff
@@ -0,0 +1,169 @@
+<template>
+  <full-page-container>
+    <template #header>
+      <header-navigator
+        title="Program detail"
+        :routeLocation="getRouteLocation()"
+        subTitle="Request to Change"
+      />
+
```

Comment:
I do not see a benefit of having this blank line right here. Please consider removing it.